src/main/java/com/iridium/iridiumskyblock/database/IslandSetting.java

```diff
@@ -22,8 +22,8 @@
     @Setter
     private String value;
 
-    public boolean getBooleanValue() {
-        return value.equalsIgnoreCase("true");
+    public boolean isAllowed() {
+        return value.equalsIgnoreCase("ALLOWED");
```

Comment:
Peaches already sent some users a version which uses true, this probably breaks everything for them.